lib/runners/processor/pmd_cpd.rb

```diff
@@ -50,12 +51,17 @@ def analyze(changes)
     end
 
     def run_analyzer
-      stdout, stderr = capture3!(analyzer_bin, *cli_options)
+      issues = []
 
-      raise_warnings(stderr)
+      languages.each do |language|
+        stdout, stderr = capture3!(analyzer_bin, *cli_options(language))
+        raise_warnings(stderr)
+        ret = construct_result(stdout)
+        issues.push(*ret)
+      end
 
       Results::Success.new(guid: guid, analyzer: analyzer).tap do |result|
-        construct_result(result, stdout)
+        issues.each { |i| result.add_issue(i) }
```

Comment:
[nitpick] Thanks to 0fdb3a62, `add_issue` can multiple issues at one time.

```suggestion
        result.add_issue(*issues)
```

Reply:
LGTM. Thank you for letting me know.
Fixed. commit 15d456b4ba1fbfb2197c404402fc500a530a5d64